### PR TITLE
[Base][Dex Trades] Add base chain dex trades about uniswap-v3

### PR DESCRIPTION
**Pull request by CsyIGy**
I realized that no one has integrated the table for Dex Trades for Base chains into Dune yet, so I wanted to be a part of this part of the build!

### CODE REVIEW

**Comment by CsyIGy** [comment]
> **CLA Assistant Lite 机器人：** 感谢您的提交，我们非常感谢。与许多开源项目一样，我们要求您签署我们的[贡献者许可协议](https://github.com/duneanalytics/spellbook/blob/main/CLA.md)，然后我们才能接受您的贡献。您只需发布与以下格式相同的 Pull 请求评论即可签署 CLA。
> 
> 我已阅读 CLA 文件并特此签署 CLA
> 
> 您可以通过在此 Pull 请求中评论**重新检查来重新触发此机器人**



> I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

**Comment by CsyIGy** [comment]
I have read the CLA Document and I hereby sign the CLA

File: models/uniswap/base/uniswap_v3_base_trades.sql [hunk 1]
```diff
@@ -0,0 +1,108 @@
+{{ config(tags=['dunesql'],
+    schema = 'uniswap_v3_base',
+    alias = alias('trades'),
+    partition_by = ['block_date'],
```
Comment (Hosuke) [inline models/uniswap/base/uniswap_v3_base_trades.sql:4]: We may add a column `block_month` similar to `block_date` and partition by `block_month`
```suggestion
    partition_by = ['block_month'],
```

**Review by Hosuke — CHANGES_REQUESTED** [review]

**Comment by CsyIGy** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by CsyIGy** [comment]
recheck

**Comment by CsyIGy** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by CsyIGy** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by CsyIGy** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request closed without merging**